cirq-core/cirq/sim/simulator_base.py

```diff
@@ -179,13 +207,43 @@ def _core_iterator(
                             continue
                         if self._ignore_measurement_results:
                             op = ops.phase_damp(1).on(*op.qubits)
-                    sim_state.axes = tuple(sim_state.qubit_map[qubit] for qubit in op.qubits)
-                    protocols.act_on(op, sim_state)
+
+                    # Go through the op's qubits and join any disparate ActOnArgs states
+                    # into a new combined state.
+                    op_args: Optional[TActOnArgs] = None
```

Comment:
Similar to the above comment, could this be simplified by initializing `op_args` to an "empty" ActOnArgs object?

Reply:
This would add a join, which is a bunch of unnecessary calculation.